### PR TITLE
Delete comment

### PR DESCRIPTION
**Pull request by ojcorrigan**
Added a button that will allow the user to delete their own comments (currently hardcoded to be cooljmessy) the comments should refresh on delete, also added form validation for if user doesn't input a comment before posting.

### CODE REVIEW

**Review by karlaferry — CHANGES_REQUESTED** [review]
A couple of comments:

- Upon submitting a comment, the comment count optimistically re-renders and increases, however it does not decrease once you've deleted a comment.
- I appreciate your form validation, however currently, if you click the submit button without clicking on the field, it submits your placeholder as the actual comment. Try to find a way to only submit when a user has actually typed something in (eg. onBlur, check value).

File: src/components/Single-article.jsx [hunk 1]
```diff
@@ -181,11 +205,21 @@ const CommentForm = ({
       >
         <label>Comment: </label>
         <textarea
+          id="commentField"
+          className={isValid ? "" : "invalid"}
```
Comment (karlaferry) [inline src/components/Single-article.jsx:209]: Lifehack: If you only have one condition in a ternary, you could use double ampersand. `!isValid && "invalid"`

**Review by karlaferry — APPROVED** [review]
Awesome! Approved :)

File: src/components/Single-article.jsx (hunk 1 above)
Comment (ojcorrigan) [reply src/components/Single-article.jsx:209]: Ha I am a fan of life hacks!

**Pull request merged**